extensions/email/src/main/java/org/mule/extension/email/api/sender/SenderOperations.java

```diff
@@ -60,7 +60,7 @@
      */
     public void send(@Connection SenderConnection connection,
                      @UseConfig SMTPConfiguration configuration,
-                     @Optional(defaultValue = PAYLOAD) EmailContent content,
```

Comment:
the EMailCOntent#body needs to default to the payload, otherwise proxy scenaio becomes complicated